overview.md

```diff
@@ -164,10 +164,27 @@ TODO
 
 ## Documentation and code exploration
 
+### Rustdoc
+
+You probably don't need to use rustdoc directly (instead going through `cargo doc`). Accepts a crate
+and generates HTML pages for its public documentation. Can also execute code blocks within the
+documentation as tests. Part of the normal Rust distribution.
+
+* [Source code](https://github.com/rust-lang/rust/tree/master/src/librustdoc) (part of the Rust
+  compiler source code)
+* Install using Rustup (see above).
+
+### Rustdoc 2
+
+In-progress reimplementation of rustdoc using the RLS as a backend (rather than compiler-internal
+libraries). Plans include JSON output, multiple pluggable "frontends" to render as different
```

Comment:
Could also link to <https://github.com/nrc/dev-tools-team/blob/master/minutes/meeting%20notes%202017-05-30.md>